sdk/appcenter/src/main/java/com/microsoft/appcenter/utils/storage/DatabaseManager.java

```diff
@@ -528,7 +528,7 @@ boolean setMaxSize(long maxStorageSizeInBytes) {
             return false;
         }
         if (maxStorageSizeInBytes != newMaxSize) {
-            AppCenterLog.warn(LOG_TAG, "Could change database size but is slightly larger than expected (next multiple of 4KB), requestedMaxSize=" +
+            AppCenterLog.info(LOG_TAG, "Could change database size but is slightly larger than expected (next multiple of 4KB), requestedMaxSize=" +
```

Comment:
The text is confusing. If i see that I might think that it didn't actually change anything and won't try to. I suggest we say `Will attempt to change database size to {size in bytes} (next multiple of 4KiB)`

Reply:
I like this suggestion!

Reply:
I like the text used in [iOS](https://github.com/Microsoft/AppCenter-SDK-Apple/pull/1125/files), we should align these anyways